src/peer/negotiator.js

```diff
@@ -407,6 +366,7 @@ class Negotiator extends EventEmitter {
         return this._pc.setLocalDescription(answer)
           .then(() => {
             logger.log('Set localDescription: answer');
+            logger.log(`Setting local description ${JSON.stringify(answer.sdp)}`);
```

Comment:
Even if user set log level to 0, `JSON.stringify()` is called.
Is it ok?

Reply:
`JSON.stringify()` is called at `_setRemoteDescription` too. So I think it is ok.